frontend/src/components/About/AboutUIModal.tsx

```diff
@@ -166,12 +132,16 @@ export const AboutUIModal: React.FC<AboutUIModalProps> = (props: AboutUIModalPro
           <TextListItem key="kiali-container-version" component="dd">
             {containerVersion!}
           </TextListItem>
-          <TextListItem key="service-mesh-name" component="dt">
-            Service Mesh
-          </TextListItem>
-          <TextListItem key="service-mesh-version" component="dd">
-            {meshVersion!}
-          </TextListItem>
+          {false && (
```

Comment:
Code section to be removed along with the `meshVersion` variable

Reply:
Code removed, and the mesh name (aka mesh version) is now displayed in the default mesh side panel.